services/core/shared/indexdb/mysql.js

```diff
@@ -186,7 +186,7 @@ const getDbInstance = async (tableName, tableConfig, connEndpoint = config.endpo
 		.whereIn(primaryKey, ids)
 		.del();
 
-	const count = async () => knex(tableName).count({ count: 'id' });
+	const count = async () => knex(tableName).count({ count: '*' });
```

Comment:
Why is this needed?

Btw, wouldn't it help to count by params?
The lines above return only all transactions in the db, without checking whether the index is finished.

Reply:
I realised now, its better to use '*'. Changing it back. Thanks